### PR TITLE
Org pr1

### PR DESCRIPTION
**Pull request by luca-pellizzon**
This is the first PR for the upcoming NewOrgChart

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:stop

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java [hunk 1]
```diff
@@ -219,22 +228,70 @@ public Organization postOrganization(Organization organization)
...
+		List<UserAccount> userAccounts = new ArrayList<>();
+
 		for (String emailAddress : emailAddresses) {
-			postUserAccountByEmailAddress(organizationId, emailAddress);
+			userAccounts.add(
+				postUserAccountByEmailAddress(organizationId, emailAddress));
 		}
+
+		if (!organizationRoleIds.isEmpty()) {
+			String[] orgRoleIds = StringUtil.split(
```
Comment (dsitu) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java:269]: Rename to `organizationRoleIds`
```suggestion
			String[] organizationRoleIds = StringUtil.split(
```

**Review by dsitu — CHANGES_REQUESTED** [review]

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java (hunk 1 above)
Comment (luca-pellizzon) [reply modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java:269]: organizationRoleIds is already defined as the parameter of the method

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**